Fix font and spacing in node selection menu

alexandercyu opened the pull request:
Fixes #139 
- Fix font and spacing between node selection menu items
- Change space between node selection menu items to 50px in tablet size

mishaschwartz commented on static/css/marble.css:
```diff
@@ -1096,7 +1096,40 @@ h6{
-  
+
+    /*Node*/
+    .margin-node-menu-item {
```
identical in all media widths

alexandercyu commented:
Moved repeated class out of media query.

mishaschwartz reviewed (changes requested):
There is currently no space between the node menu items on `sm` size

alexandercyu commented:
> There is currently no space between the node menu items on `sm` size

Fixed the spacing between node menu items on sm size.

Cleaned up" >=576px" media query.

mishaschwartz reviewed (approved):
This looks good.

It's a little difficult to review all the of the code cleanup since a lot of that doesn't seem related to this PR but as far as I can tell by looking at the UI it doesn't seem to break anything.

alexandercyu commented:
> This looks good.
> 
> It's a little difficult to review all the of the code cleanup since a lot of that doesn't seem related to this PR but as far as I can tell by looking at the UI it doesn't seem to break anything.

The cleanup was mostly to get rid of duplicates in the " >=576px" media query.  The differences I remember are the space between the node page heading and value statement, and the padding in the node services cards.
